Tools/Scripts/webkitpy/layout_tests/controllers/manager.py

```diff
@@ -733,10 +733,15 @@ def _save_json_files(self, summarized_results, initial_results):
 
     def _copy_results_html_file(self, filename, destination_path):
         base_dir = self._port.path_from_webkit_base('LayoutTests', 'fast', 'harness')
+        backup_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../../../', 'LayoutTests', 'fast', 'harness'))
         results_file = self._filesystem.join(base_dir, filename)
+        backup_file = self._filesystem.join(backup_dir, filename)
         # Note that the results.html template file won't exist when we're using a MockFileSystem during unit tests,
         # so make sure it exists before we try to copy it.
-        if self._filesystem.exists(results_file):
+        if not self._filesystem.exists(results_file):
+            if self._filesystem.exists(backup_file):
```

Comment:
Rather than this nested check, we should do:
```
if not self._filesystem.exists(results_file):
    backup_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../../../', 'LayoutTests', 'fast', 'harness'))
    results_file = self._filesystem.join(backup_dir, filename)
if self._filesystem.exists(results_file):
    self._filesystem.copyfile(results_file, destination_path)
```

I think that will be easier to understand, rather than this nested if statement thing

Reply:
> Rather than this nested check, we should do:
> 
> ```
> if not self._filesystem.exists(results_file):
>     backup_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../../../', 'LayoutTests', 'fast', 'harness'))
>     results_file = self._filesystem.join(backup_dir, filename)
> if self._filesystem.exists(results_file):
>     self._filesystem.copyfile(results_file, destination_path)
> ```
> 
> I think that will be easier to understand, rather than this nested if statement thing

I agree that is easier to read. Making that change.